packages/amplify-category-hosting/commands/hosting/remove.js

```diff
@@ -1,15 +1,84 @@
+const fs = require('fs-extra');
+const path = require('path');
+const inquirer = require('inquirer');
+
 const subcommand = 'remove';
 const category = 'hosting';
 
+const pluginManifest = require('../../amplify-plugin.json');
+
 module.exports = {
   name: subcommand,
   run: async context => {
-    const { amplify, parameters } = context;
-    const resourceName = parameters.first;
+    const { amplify, input } = context;
+
+    const inputResourceName = input.subCommands && input.subCommands.length > 0 ? input.subCommands[0] : undefined;
+    const resourceName = await chooseResource(context, inputResourceName);
```

Comment:
See the above comments. 
The design of the plugin from the start is that it can be extended to handle more than one service. And this update also allows other plugins with pluginName "hosting" to have other services.